packages/form/src/helpers/create-survey.ts

```diff
@@ -20,29 +13,32 @@ import {
 /**
  * Calls the Survey123 create API with the given parameters
  * @param {ISurvey123CreateParams} params
- * @param {UserSession} authentication
  * @param {string} [survey123Url=https://survey123.arcgis.com] An optional, Survey123 base URL override
  * @throws Will throw if the Survey123 API returns an error response
  * @returns {Promise<ISurvey123CreateResult>}
  */
 export function createSurvey(
   params: ISurvey123CreateParams,
-  authentication: UserSession,
   survey123Url = "https://survey123.arcgis.com"
 ): Promise<ISurvey123CreateResult> {
   const createUrl = `${survey123Url}/api/survey/create`;
-  const requestOptions = {
-    httpMethod: "POST",
-    authentication,
-    params: {
-      f: "json",
-      ...params
-    }
-  } as IRequestOptions;
-  return request(createUrl, requestOptions).then(
-    (
-      response: ISurvey123CreateSuccess | ISurvey123CreateError
-    ): ISurvey123CreateResult => {
+  const ro = {
+    credentials: "same-origin" as RequestCredentials,
+    method: "POST",
+    body: encodeFormData(
+      {
+        f: "json",
+        ...params
+      },
+      true
+    )
+  };
+  // Using @esri/arcgis-request "request" method was resulting in a 404 for
+  // a CORS preflight request related to this request, but calling fetch directly
+  // circumvents the issue.
+  return fetch(createUrl, ro)
```

Comment:
`request` provides token refreshing (and other things) in addition to its fetch. Is there any way that you can work with or improve `request`?